libcontainer/user/lookup.go

```diff
@@ -6,11 +6,12 @@ import (
 )
 
 var (
-	// The current operating system does not provide the required data for user lookups.
+	// ErrUnsupported: The current operating system does not provide the required data for user lookups.
```

Comment:
This really should be reworded slightly so its not just eliminating the lint warning but also making it read well e.g.
```// ErrUnsupported is returned if the current operating system does not provide the required data for user lookups.```